@xen-orchestra/build-template/src/index.spec.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-env jest */
+import buildTemplate from '.'
+
+it('builds a template', () => {
+  const replacer = buildTemplate('{name}_\\{name}_\\\\{name}_{id}_%_FOO', {
+    '{name}': obj => obj.name,
+    '{id}': 1235,
+    '%': (_, i) => i,
+  })
+  expect(replacer({ name: 'toto' }, 5)).toBe('toto_{name}_\\toto_1235_5_FOO')
```

Comment:
`{ name: 'foo' }` :)

Reply:
We already have foo. It'll be ambiguous.

Reply:
`'bar'` then. My point is don't use "toto".